### PR TITLE
Set up OpenCensus in integration test

### PR DESCRIPTION
**Pull request by zoercai**
Please try this on your local

### CODE REVIEW

File: sdks/java/io/google-cloud-platform/build.gradle [hunk 1]
```diff
@@ -145,6 +145,9 @@ dependencies {
   testCompile library.java.powermock
   testCompile library.java.powermock_mockito
   testCompile library.java.joda_time
+  testCompile 'io.opencensus:opencensus-api:0.28.3'
```
Comment (thiagotnunes) [inline sdks/java/io/google-cloud-platform/build.gradle:148]: @zoercai Thanks for putting this up. I don't think we should actually commit it, but it is nice to have an example so that we can set it up on the apps that are using the connector.

Do you want to move the instance changes into their own PR and we can merge that one?

Comment (zoercai) [reply sdks/java/io/google-cloud-platform/build.gradle:148]: This is merging into load-test, and the instance changes are just the ones I pulled from `spanner-cdc`

Comment (thiagotnunes) [reply sdks/java/io/google-cloud-platform/build.gradle:148]: Ah yes, thanks, that makes sense, sorry I missed it.

**Review by thiagotnunes — APPROVED** [review]
